third_party/istio-1.1-rc/download-istio.sh

```diff
@@ -0,0 +1,77 @@
+# Download and unpack Istio
+ISTIO_VERSION=1.1.0-rc.4
+DOWNLOAD_URL=https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-linux.tar.gz
+
+wget $DOWNLOAD_URL
+tar xzf istio-${ISTIO_VERSION}-linux.tar.gz
+cd istio-${ISTIO_VERSION}
+
+# Create CRDs template
+helm template --namespace=istio-system \
+  install/kubernetes/helm/istio-init \
+  > ../istio-crds.yaml
+
+# Create a custom cluster local gateway, based on the Istio custom-gateway template.
+helm template --namespace=istio-system \
```

Comment:
We also need key `gateways.istio-ingressgateway.sds.enabled=true` in order to support https://github.com/knative/serving/issues/1767 and https://github.com/knative/serving/issues/3052.